core/store/migrate/migrations/0090_ocr2_relay.sql

```diff
@@ -0,0 +1,28 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE offchainreporting2_oracle_specs
+    ADD COLUMN relay text,
+    ADD COLUMN relay_config JSONB,
+    DROP COLUMN evm_chain_id,
+    DROP CONSTRAINT chk_contract_address_length;
+ALTER TABLE offchainreporting2_oracle_specs
+    RENAME COLUMN contract_address TO contract_id;
+ALTER TABLE offchainreporting2_oracle_specs
+    RENAME COLUMN transmitter_address TO transmitter_id;
```

Comment:
Do we also need to cast to `text` here?

Reply:
This really feels like it should be part of the relay config

Reply:
transmitterID actually references a key in the core keystore, same as the OCR key bundle ID and is common to all relays so imo makes sense outside the relay config

Reply:
Yup, that makes sense to me